crates/context/config/src/client/config.rs

```diff
@@ -61,6 +62,16 @@ pub struct ClientLocalSigner {
     pub rpc_url: Url,
     #[serde(flatten)]
     pub credentials: Credentials,
+    #[serde(flatten)]
+    pub near_credentials: Option<NearCredentials>,
+    #[serde(flatten)]
+    pub starknet_credentials: Option<StarknetCredentials>,
+    #[serde(flatten)]
+    pub ethereum_credentials: Option<EthereumCredentials>,
+    #[serde(flatten)]
+    pub icp_credentials: Option<IcpCredentials>,
+    #[serde(flatten)]
+    pub stellar_credentials: Option<StellarCredentials>,
```

Comment:
```suggestion
    #[serde(flatten)]
    pub credentials: serde_json::Value,
```

you can equivalently get rid of `Credentials` and `RawCredentials` below this